### DISCUSSION TITLE
Cannot installed gatsby-remark-reading-time plugin

### DISCUSSION THREAD

**Discussion by tashasyarifah** [gatsbyjs/gatsby#34622] [Help] [answered]
Hi everyone, 

I'm Tasha and creating my website by following tutorials on their official website. Right now, I want to add some features one of them is reading time. I cannot install the `gatsby-remark-reading-time` plugin. 

Here's what my terminal says.
![Screenshot of Tasha's terminal about her error](https://user-images.githubusercontent.com/66051012/151692033-aa7c1829-5c69-4183-8f14-dfa8c270dccf.png)

If anyone knows what it means and how to solve it, please let me know.

**Comment by alex-semenyuk** [comment, marked as answer]
Seems similar to this one https://github.com/gatsbyjs/gatsby/issues/28925.
 Try `npm install --legacy-peer-deps` as mentioned there.

**Reply by tashasyarifah** [reply]
It works! Thanks, Alex.

**Reply by Ayushtig** [reply]
how it worked its not working for me

**Reply by Ayushtig** [reply]
Please help me in this.
